introduction.md

```diff
@@ -0,0 +1 @@
+* Hi, it's Elan. Trying this again from the start to make sure I get this absolutely correct.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.